indexer/feature_altitude.hpp

```diff
@@ -81,46 +83,34 @@ class Altitudes
   template <class TSink>
   void Serialize(TAltitude minAltitude, TSink & sink) const
   {
-    CHECK(!m_altitudes.empty(), ());
-
-    WriteVarInt(sink, static_cast<int32_t>(m_altitudes[0]) - static_cast<int32_t>(minAltitude));
-    for (size_t i = 1; i < m_altitudes.size(); ++i)
-    {
-      WriteVarInt(sink,
-                  static_cast<int32_t>(m_altitudes[i]) - static_cast<int32_t>(m_altitudes[i - 1]));
-    }
+    vector<uint32_t> deviations;
+    PrepareSerializationData(minAltitude, deviations);
+
+    BitWriter<TSink> bits(sink);
+    for (auto const d : deviations)
```

Comment:
I got rid of vector of deltas but did it in another way. Please have a look at the code.